image/php-fpm/php-fpm.conf

```diff
@@ -22,7 +22,7 @@
 ; Pid file
 ; Note: the default prefix is /var
 ; Default Value: none
-pid = /run/php5-fpm.pid
+pid = /var/run/php5-fpm.pid
```

Comment:
Whatever the debian/ubuntu defaults for this.  I'm easy.  Just want to make sure we don't deviate from their standards so that we remain convention and tool compatible.